src/config/strategies/setup-user.ts

```diff
@@ -34,7 +34,6 @@ const PROFILE_ATTRS = {
   'urn:oid:2.16.840.1.113730.3.1.241': 'displayName',
   'urn:oid:1.3.6.1.4.1.5923.1.1.1.9': 'scopedAffiliation',
   'urn:oid:0.9.2342.19200300.100.1.3': 'email',
-  'urn:oid:2.16.840.1.113730.3.1.3': 'employeeNumber',
```

Comment:
I can't vouch for whether this attr is needed or used. Assuming its not needed, than this should be good